x/emissions/proto/emissions/v1/types.proto

```diff
@@ -244,6 +246,7 @@ message ForecastElement {
       [(gogoproto.customtype) = "github.com/allora-network/allora-chain/math.Dec", (gogoproto.nullable) = false];
   bytes extra_data = 4;
   string proof = 5;
+  bytes signature = 6;
```

Comment:
Just to ensure we're on the same page:
Each forecast is a set of predictions => each forecaster is making a `Forecast` which contains a list of these signature elements.

Therefore, could we not have this signature on the Forecast instead of forecast element?
If so => way fewer signatures have to be checked

Or, is there some technical limitation preventing that?